internal/grill/runner.go

```diff
@@ -162,3 +225,16 @@ func (suite *TestSuite) Run(ctx TestContext) error {
 
 	return nil
 }
+
+const digits = "0123456789"
+
+// makeTestBreak generates a randomized line used to separate output of
+// individual commands in a suite. Randomized element is added to reduce
+// the change of the string occurring in the test output itself.
+func makeTestBreak() string {
+	b := make([]byte, 8)
+	for i := range b {
+		b[i] = digits[rand.Intn(len(digits))]
+	}
+	return "GRILL" + string(b) + ":"
+}
```

Comment:
This is fine, but you can also do this fwiw
```
b := make([]byte, 8)
rand.Read(b)
return hex.EncodeToString(b)
```

Reply:
Much nicer; fixed